src/irmin-pack/unix/gc_worker.ml

```diff
@@ -109,7 +109,7 @@ module Make (Args : Gc_args.S) = struct
     let path = Irmin_pack.Layout.V3.suffix ~root ~generation in
     Ao.create_rw_exn ~path
 
-  let run ~generation root commit_key =
```

Comment:
I just noticed that the last parameter here has the wrong name. It should be `new_prefix_start_offset`